.packit.yaml

```diff
@@ -0,0 +1,14 @@
+downstream_package_name: python-urlgrabber
+jobs:
+- job: copr_build
+  metadata:
+    targets:
+    - fedora-30-x86_64
+    - fedora-31-x86_64
+    - fedora-rawhide-x86_64
```

Comment:
What would be required if I wanted to add more distributions to this packit setup?

Reply:
For adding new targets, only new value in metadata targets will do it. However, it will only work with distributions supported by copr.